Runtime/Scripts/EdgeEventsUtil.cs

```diff
@@ -350,8 +362,8 @@ public static IEnumerator RespondToLatencyRequest(ConnectionDetails connectionDe
       {
         throw new NullReferenceException("edgeEventsConnection is null, Make sure you have a running edge events connection or that you are passing the correct arguments.");
       }
-      DeviceInfoDynamic deviceInfoDynamic = EdgeEventsManager.GetDynamicInfo(connectionDetails.matchingEngine);
-      AppPort appPort = EdgeEventsManager.GetLatencyTestPort(connectionDetails.mobiledgexManager, latencyPort);
+      DeviceInfoDynamic deviceInfoDynamic = EdgeEventsManager.GetDeviceDynamicInfo(connectionDetails.matchingEngine);
```

Comment:
DeviceInfoDynamic ?

Reply:
This is coming from C#
<img width="436" alt="Screen Shot 2022-03-11 at 10 11 24 AM" src="https://user-images.githubusercontent.com/24863504/157925531-644cf88f-46c9-47e2-8b9b-c691efb4a4de.png">